File: demo-vk-rest-scraping.java/src/main/java/demo/model/CountriesList.java
package demo.model;

import java.util.List;

public class CountriesList {

    private Integer countriesCount;
    private List<Country> countries;

    public CountriesList() {
    }

    public CountriesList(List<Country> countries) {
        this.countries = countries;
        this.countriesCount = countries.size();
    }

    public List<Country> getCountries() {
        return countries;
    }

    public void setCountries(List<Country> countries) {
        this.countries = countries;
        this.countriesCount = countries.size();
    }

    public Integer getCountriesCount() {
        return countriesCount;
    }
}

File: demo-vk-rest-scraping.kotlin/src/main/kotlin/demo/model/CountriesList.kt
package demo.model

class CountriesList {

    var countriesCount: Int? = null
        private set

    var countries: List<Country>? = null
        set(countries) {
            field = countries
            this.countriesCount = countries!!.size
        }

    constructor() {
    }

    constructor(countries: List<Country>) {
        this.countries = countries
        this.countriesCount = countries.size
    }
}
